public/scripts/extensions/gallery/index.js

```diff
@@ -406,7 +407,7 @@ SlashCommandParser.addCommandObject(SlashCommand.fromProps({ name: 'show-gallery
         showCharGallery();
         return '';
     },
-    helpString: 'Shows the gallery.',
+    helpString: translate('Shows the gallery.'),
```

Comment:
Let's maybe handle slash commands some other time. I'm not sure what would be the ultimate way to approach them.

Reply:
fixed in https://github.com/SillyTavern/SillyTavern/commit/3e5f98223d0c45ea1a94f793bcc075c84ef470cd